spec_test.go

```diff
@@ -109,6 +109,9 @@ func TestSpecYamlEncoding(t *testing.T) {
 					ID: "fyz88",
 				},
 			},
+			Template: clustertprspec.Template{
+				Version: "0.0.1",
+			},
```

Comment:
This looks too generic to me which I expressed in already merged clustertpr PR.

Reply:
Unrelated to this PR.